lib/acts_as_scrubbable/scrubbable.rb

```diff
@@ -1,36 +1,35 @@
 module ActsAsScrubbable
   module Scrubbable
-
-
     def scrubbable?
       false
     end
 
+    def acts_as_scrubbable(scrub_type=:scrub, *scrubbable_fields, **mapped_fields)
```

Comment:
i don't think you can have default values to the left of var-args?

Reply:
You can! it just isn't respected. Was having trouble finding a good way to fit in the scrub_type while not creating too breaking of a change. I think the approach you listed at the top is better (kwargs all the way, arrays of symbols or kwargs as values